README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
 
 ISO/IEC 8652 is officially maintained by ISO/IEC JTC 1/SC 22/WG 9. Comments from the general public are handled by WG 9's Ada Rapporteur Group (ARG). The ARG is responsible for addressing these comments, which often concern possible enhancements or errors in the Ada standard. The ARG considers these enhancements or errors for incorporation into future versions of the Ada standard.
 
-## A new way to engage the Ada user community
+## Ada Interpretations (AIs)
 
-Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements.  
+The ARG performs most of its work in terms of Ada Interpretations (AIs), which are documents that describe an identified issue with the Ada Reference Manual, or which propose enhancements to the Ada programming language.  Such issues may be identified within the ARG itself, or by the wider Ada community.
+
+At ARG meetings, the ARG will deliberate on active AIs, and vote on the progression of each AI, ultimately towards acceptance, or no action. Discussion and votes are available via the [ARG minutes](http://ada-auth.org/arg-minutes.html).
+
+AIs that are being or have been deliberated on by the ARG can be found [here](https://arg.adaic.org/ada-interpretations).
+
+As the ARG deliberates on open AIs, AIs will be updated with changes and additional discussion, or may lead to the establishment of a Language Study Group if more intensive development work is required.
+
+At the present time, AIs are being stored in the Rich Text Format, with an aim to eventually develop tooling to easily convert it into many formats. RTF was selected due to its maturity, stability, and generality. The ARG will continue to review the AI formats in the future.
+
+An AI template can be found in the root of this repository.
```

Comment:
Somewhere it would make sense to add a link to the material about creating proper AIs (http://www.ada-auth.org/ai-form.html). It's linked from inside the template, but I wouldn't expect most people to look in there until they're committed to creating an AI. For people just considering getting involved, the details could be important as to whether they decide to undertake the project at all.

Reply:
Added a line about this above the one for the template.